python/iceberg/api/expressions/strict_metrics_evaluator.py

```diff
@@ -15,27 +15,31 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import threading
+
 from .expressions import Expressions, ExpressionVisitors
 from ..expressions.binder import Binder
 from ..types import Conversions
 
 
 class StrictMetricsEvaluator(object):
 
+    def visitor(self):
+        if not hasattr(self.thread_local_data, "visitors"):
+            self.thread_local_data.visitors = StrictMetricsEvaluator.MetricsEvalVisitor(self.expr,
+                                                                                                          self.schema,
```

Comment:
Nit: Formatting looks weird here.

Reply:
I'll clean this up in a future PR, I had refactored and didn't line the indent back up.